assets/src/components/dup/dup_screen_page.tsx

```diff
@@ -37,7 +37,7 @@ const ScreenPage = ({
 }: {
   screenContainer: React.ComponentType;
 }): JSX.Element =>
-  isDup() ? (
+  isDUP() ? (
```

Comment:
Yeah, my bad, this is an artifact of my initially merging the check into a new `isOFM` function, and then needing to go back and have separate `isDUP` and `isTriptych` functions. In the process I accidentally renamed the original `isDup` to `isDUP`. 😵‍💫

I'll restore this function's original name.

Reply:
I checked the latest updates, and it still looks like the changes are there, so we might be talking about different things.

Since this is a DUP v1 file, I'm wondering why the change to the function name needed to be made? We don't have any DUP v1s in the world anymore and I'm not concerned about keeping it maintained, but maybe that's a philosophically bad approach?